2.4/root/usr/share/container-scripts/mongodb/initiate_replica.sh

```diff
@@ -66,7 +66,7 @@ echo "=> Waiting for the new PRIMARY to be elected ..."
 mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --quiet --host ${mongo_node} --eval "var done=false;while(done==false){var members=rs.status().members;for(i=0;i<members.length;i++){if(members[i].stateStr=='PRIMARY' && members[i].name!='$(mongo_addr)'){done=true}};sleep(500)};" &>/dev/null
 
 # Remove the initialization container MongoDB from cluster and shutdown
-echo "=> The new PRIMARY member is $(mongo_primary_member_addr), shutting down current member ..."
+echo "=> The new PRIMARY member elected, shutting down current member ..."
```

Comment:
"**A** new PRIMARY member **was elected**, shutting down current member ..."

Still, "current member" is really strange... but we don't need to bike shed on that.